### ISSUE TITLE
More defaults

### ISSUE THREAD

**Issue by alitvak69** [GROG/ansible-role-sudo#10]
I added if few lines to your defaults handling if  you care. Needed this for default options

default/main.yml

``` yml
#Defaults for Commands
sudo_default_command_options: []
```

templates/etc-sudoers.d-defaults_template.j2

``` jinja
{% for entry in sudo_default_command_options %}
{{ entry.default_command_opt }} {{ entry.command_opt_val  }}
{% endfor %}
```

Example in group_vars/all.yml 

``` yml
sudo_default_command_options:
  - default_command_opt: Defaults!LOG_VIEWERS
    command_opt_val: "noexec"
  - default_command_opt: Defaults!CRM_COMMANDS
    command_opt_val: `"!requiretty" 
```

**Comment by groggemans** [comment]
Thank you for the patch! I'll probably add it tomorrow.

**Comment by groggemans** [comment]
Implemented in #11!

I used a different implementation so all defaults are in the same list, you can see an example in the README.

Thank you for your contribution!
